airflow/providers/google/cloud/transfers/presto_to_gcs.py

```diff
@@ -150,12 +150,12 @@ class PrestoToGCSOperator(BaseSQLToGCSOperator):
 
     type_map = {
         "BOOLEAN": "BOOL",
-        "TINYINT": "INT64",
-        "SMALLINT": "INT64",
-        "INTEGER": "INT64",
-        "BIGINT": "INT64",
-        "REAL": "FLOAT64",
-        "DOUBLE": "FLOAT64",
+        "TINYINT": "INTEGER",
```

Comment:
Can you give more context to what is the bug?

I believe that the goal was to make the files in GCS to be compatible with BigQuery (to allow uploading to BigQuery right after)
These are all aliases:
`INT, SMALLINT, INTEGER, BIGINT, TINYINT, and BYTEINT are aliases for INT64.
`https://cloud.google.com/bigquery/docs/reference/standard-sql/data-types